app/src/community/res/layout/development_activity_login.xml

```diff
@@ -93,6 +94,7 @@
                     android:id="@+id/community_login_screen_passcode_edittext"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:autofillHints="password"
```

Comment:
same here... string resource?

Reply:
No this corresponds to a constant that is defined in the Android `View` class [here](https://developer.android.com/reference/android/view/View#AUTOFILL_HINT_PASSWORD).